x-pack/plugins/xpack_main/server/lib/telemetry/monitoring/get_high_level_stats.js

```diff
@@ -212,7 +226,7 @@ export function fetchHighLevelStats(server, callCluster, clusterUuids, start, en
         type: `${product}_stats`,
         filters: [
           { terms: { cluster_uuid: clusterUuids } },
-          { exists: { field: 'kibana_stats.usage.index' } }
+          ...getFiltersForStackProduct(product)
```

Comment:
`fetchHighLevelStats` is called for other indices like logstash, and so we can't include a kibana specific filter.